pages/common/npm.md

```diff
@@ -27,6 +27,10 @@
 
```

Comment:
Please change this description to `List of tree of installed modules referenced in package.json`

Reply:
Shouldn't `package.json` be in backticks?

Reply:
Backticks are only for commands, not filenames like these. I also checked the yarn.md page. It also refers to package.json simply without any backticks.

Reply:
Ah, ok 😺